### PR TITLE
No third party plugins version

### PR DESCRIPTION
**Pull request by MyCodingSucks**
Redo some works. So I closed the old one.
I just do directly on the website, so maybe some files are **out of date** compared with bitbrain's. (Like 404 page)

---

It just doesn't have those unavailable third-party plugins. I don't know whether those I didn't delete are working or not, but anyway, it can be used.

Also, as I don't know too much about programming, the place I changed maybe look very different between mine and the original one. So I think it is better to create a new branch, but I only find the "master" section when I try to open a pull request. And also it would be better if someone can improve it as the style is too different. ~~Git is too complex for me~~.

Finally, sorry for my bad programming style.

### CODE REVIEW

File: README.md
Comment (bitbrain) [inline README.md:6]: why did you remove this? :(

File: README.md
Comment (bitbrain) [inline README.md:4]: also revert this :)

File: README.md
Comment (bitbrain) [inline README.md:1]: we wanna keep the logo

**Comment by bitbrain** [comment]
@MyCodingSucks great PR from the first looks. However, you have removed a lot of stuff that should not be removed. Try to bring back everything that is not directly related to your changes, please.

**Review by bitbrain — CHANGES_REQUESTED** [review]
see comments

File: README.md
Comment (MyCodingSucks) [reply README.md:6]: Sorry, I upload the wrong one. I created three or four README and I didn't check it before and after I uploaded. I will upload the new one, which I just add something at the bottom of your original README.

Sorry for this mistake.

**Comment by MyCodingSucks** [comment]
> @MyCodingSucks great PR from the first looks. However, you have removed a lot of stuff that should not be removed. Try to bring back everything that is not directly related to your changes, please.

@bitbrain Now it should be back, I think.

**Comment by bitbrain** [comment]
@MyCodingSucks sorry, I did not have time to migrate the new deployment pipeline. Before that isn't done we cannot merge this, I am afraid!

**Comment by MyCodingSucks** [comment]
> @MyCodingSucks sorry, I did not have time to migrate the new deployment pipeline. Before that isn't done we cannot merge this, I am afraid!

Sorry for my late reply. I just went back home from school few days ago, and I have to deal with the route in my home. Anyway, you can do it whenever you want.

**Comment by bitbrain** [comment]
Hello @MyCodingSucks - unfortunately, I will close this, as we do not require this any longer.

**Pull request closed without merging**
